src/components/QuillEditor/Buttons/EditorButtons.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/display-name */
```

Comment:
no need to disable it, You can just add static displayName in the same way we have in:
https://github.com/appsome-solutions/personashare-fe/blob/master/src/components/FormikFields/FormikInput/FormikInput.tsx#L39